.github/workflows/ruby.yml

```diff
@@ -89,7 +89,7 @@ jobs:
       with:
         persist-credentials: false
     - uses: actions/setup-node@v3
-    - run: npm -g install yalc ${{ matrix.js_package_manager.installer }}
+    - run: sudo npm -g install yalc ${{ matrix.js_package_manager.installer }}
```

Comment:
Seems weird that we have to use `sudo` but not that concerned since this in a temp CI runner 😁

(I figure it might be related to supporting old versions of Node and what not)

Reply:
The `sudo` is to resolve a `EACCES: permission denied` error that I was getting.

The other option I saw in my search was `--unsafe-perm`, but that's apparently obsolete.